.github/workflows/continuous-integration.yml

```diff
@@ -131,8 +131,10 @@ jobs:
     - name: yarn install and test
       run: |
         cd src/rest-server
+        cp -rf ../database-controller/sdk openpaidbsdk
```

Comment:
could you add a `preinstall` script in `rest-server/package.json` and run `yarn preinstall` instead?

Reply:
Done.